src/test/java/com/google/cloud/spanner/pgadapter/AbstractMockServerTest.java

```diff
@@ -213,6 +224,28 @@ public static void startMockSpannerAndPgAdapterServers() throws Exception {
             .addService(mockSpanner)
             .addService(mockDatabaseAdmin)
             .addService(mockOperationsService)
+            .intercept(
+                new ServerInterceptor() {
+                  @Override
+                  public <ReqT, RespT> Listener<ReqT> interceptCall(
+                      ServerCall<ReqT, RespT> serverCall,
+                      Metadata metadata,
+                      ServerCallHandler<ReqT, RespT> serverCallHandler) {
+
+                    if (SpannerGrpc.getExecuteStreamingSqlMethod()
+                        .getFullMethodName()
+                        .equals(serverCall.getMethodDescriptor().getFullMethodName())) {
+                      String userAgent =
+                          metadata.get(
+                              Metadata.Key.of(
+                                  "x-goog-api-client", Metadata.ASCII_STRING_MARSHALLER));
```

Comment:
The client library token header that includes the value is named `x-goog-api-client`.